app/models/deploy.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 class Deploy < ActiveRecord::Base
+  include AuditLog
```

Comment:
this will fail when the plugin is not loaded, so needs to be in `plugins/audit/app/decorators/deploy_decorator.rb`

```Ruby
Deploy.include AuditLog
```